lib/emque/producing/message/message_with_changeset.rb

```diff
@@ -30,15 +46,31 @@ class ChangesPayloadGenerator
       def initialize(opts = {})
         @original = opts.fetch(:original, {}) || {}
         @updated = opts.fetch(:updated, {}) || {}
+        @translated_attrs = opts.fetch(:translated_attrs, {}) || {}
```

Comment:
Roger that on syntax. As far as the additional `|| {}`, I think it's necessary to account for this:
```ruby
ChangesPayloadGenerator.new(
  {
      :original => { ... },
      :updated => { ... },
      :translated_attrs => nil
  }
).execute

# then....
@translated_attrs = opts.fetch(:translated_attrs) { {} } 
@translated_attrs 
=> nil
```

And I think I'd prefer to just have an empty hash being available in the rest of the class rather than having to guard against nil. Also, this was the precedent set by the preceding instance vars, and all tests fail if the change is made to those, so I think it's a consistency that's worth staying with. Thoughts?